src/test/regress/sql/multi_test_helpers.sql

```diff
@@ -169,7 +169,7 @@ $$ LANGUAGE plpgsql;
 
 -- Introduce a function that waits until all cleanup records are deleted, for testing purposes
 CREATE OR REPLACE FUNCTION wait_for_resource_cleanup() RETURNS void
-SET client_min_messages TO WARNING
+SET client_min_messages TO ERROR
```

Comment:
Now that you added this, you should be able to remove `SET client_min_messages TO ERROR` in lots of places where you added it in this PR.